azure-pipelines.yml

```diff
@@ -0,0 +1,57 @@
+jobs:
+
+- job:
+  strategy:
+    matrix:
+      Mojave-Xcode-10.2.1:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '10.2.1'
+      Mojave-Xcode-11.0:
+        IMAGE_POOL: 'macOS-10.14'
+        XCODE_VERSION: '11'
+  pool:
+    vmImage: $(IMAGE_POOL)
+  
```

Comment:
There are trailing and unnecessary whitespaces in this file.  Can you remove them?

![image](https://user-images.githubusercontent.com/466104/62446365-d33a4480-b762-11e9-9b92-21774403fcfc.png)